overview.md

```diff
@@ -4,81 +4,86 @@ css_id: overview
 ---
 
 ## Overview
+A quick look at what TUF does and how it works
```

Comment:
This sentence doesn't end with a period.

Reply:
True. I was seeing this as a subhead, which is why I didn't add the period. Its fixed now